pkg/disttask/framework/taskexecutor/task_executor.go

```diff
@@ -532,8 +533,12 @@ func (e *BaseTaskExecutor) onError(err error) {
 	if err == nil {
 		return
 	}
-	err = errors.Trace(err)
-	e.logger.Error("onError", zap.Error(err), zap.Stack("stack"))
+
+	if errors.HasStack(err) {
```

Comment:
why check it?

Reply:
we don't need to print the plain error stack. There's already zap.Error(err)